config/peerpods/podvm/gcp-podvm-image-handler.sh

```diff
@@ -254,6 +268,7 @@ if [ "$1" = "--" ]; then
 else
   while getopts "cCRh" opt; do
     verify_vars
+    login_to_gcp
```

Comment:
Is it needed to login again for every opt on the command line ?
Same remark for `verify_vars` even it predates this PR.